pkg/gameservers/portallocator_test.go

```diff
@@ -120,7 +120,7 @@ func TestPortAllocatorAllocate(t *testing.T) {
 			// ports between 10 and 20
 			for i := 10; i <= 20; i++ {
 				var p int32
-				gs, err := pa.Allocate(fixture.DeepCopy()) // nolint: vetshadow
+				gs, err := pa.Allocate(fixture.DeepCopy())
```

Comment:
Just for curiosity - does this mean we aren't checking shadowing, or just that this vet shadow check is a bit more sophisticated?

Reply:
Apparently it's true by default.

```
linters-settings:
  govet:
    check-shadowing: true
```

Reply:
So I think it's more sophisticated.